src/main/java/edu/cornell/kfs/coa/businessobject/CuObjectCodeActivationGlobalSearch.java

```diff
@@ -0,0 +1,36 @@
+package edu.cornell.kfs.coa.businessobject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.coa.businessobject.ObjectCode;
+import org.kuali.kfs.gl.businessobject.Balance;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class CuObjectCodeActivationGlobalSearch extends ObjectCode {
+    private static final long serialVersionUID = 7461668856007571783L;
+    
+    protected List<Balance> balances;
+    
+    public CuObjectCodeActivationGlobalSearch() {
+        super();
+        balances = new ArrayList<Balance>();
+    }
+
+    public List<Balance> getBalances() {
+        return balances;
+    }
+
+    public void setBalances(List<Balance> balances) {
+        this.balances = balances;
+    }
+
+    public KualiDecimal getCalculatedBalance() {
```

Comment:
Could this property and/or the "balances" property be added to the extension object instead, to simplify the configuration needed to implement this? Or should this new calculated balance only be specific to the new Object Code Activation Global process?

Reply:
at this point the balance in should only be in the lookup for Object Code Activation Global.  Can I setup a new extension just for this new search?

Reply:
Chad and I discussed this, the transient BO option will be more complicated than the approach I took, so we are going to use this solution for now.